src/components/organisms/BannerAdmin.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useCallback, useRef } from "react";
+
+import { makeUpdateBanner } from "api/bannerAdmin";
+
+import { currentVenueSelector } from "utils/selectors";
+
+import { useSelector } from "hooks/useSelector";
+
+export const BannerAdmin: React.FC = () => {
+  const currentVenue = useSelector(currentVenueSelector);
+  const venueId = currentVenue?.id;
+  const existingBannerMessage = currentVenue?.bannerMessage ?? "";
+
+  const inputFieldRef = useRef<HTMLInputElement>(null);
+  const [error, setError] = useState<string | null>();
+
+  const handleInputChange = useCallback(() => {
+    setError(null);
+  }, []);
+
+  const updateBannerInFirestore = useCallback(
+    (msg?: string) => {
+      if (!venueId) return;
+
+      makeUpdateBanner(venueId, (errorMsg) => setError(errorMsg))(msg);
+    },
+    [venueId]
+  );
+
+  const saveBanner = useCallback(() => {
+    if (!inputFieldRef.current) return;
+
+    updateBannerInFirestore(inputFieldRef.current.value);
+
+    return false;
```

Comment:
return false?

Reply:
My guess is that it's intent was to prevent the default action from the `<button>` `onClick` handler, but that doesn't work in React:

- https://reactjs.org/docs/handling-events.html
  - > Another difference is that you cannot `return false` to prevent default behavior in React. You must call `preventDefault` explicitly.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1080/commits/d712013d6dc59ef2a8b177f0050bcf5eb30510ce